glotaran/parameter/parameters.py

```diff
@@ -157,6 +157,10 @@ def from_dataframe(cls, df: pd.DataFrame, source: str = "DataFrame") -> Paramete
 
         .. # noqa: D414
         """
+        df.rename(
```

Comment:
I don't like `parameter` package be aware of this, this should be done within plugins.

Reply:
I included a comment about this in my review [here](https://github.com/glotaran/pyglotaran/pull/1174#pullrequestreview-1177935323).

Reply:
Fixed in [6aa60e6](https://github.com/glotaran/pyglotaran/pull/1174/commits/6aa60e6b6f51b1d8b6b7a7d03364d58c2fc6dffb)